### DISCUSSION TITLE
Finding Alexa Skill IDs and/or provide follow-up command

### DISCUSSION THREAD

**Discussion by JvdMaat** [alandtse/alexa_media_player#1545] [Q&A] [answered]
Could anyone elaborate on:
You can find skill ID's by inspecting the JSON when navigating to https://alexa.amazon.com/ and clicking on skills.
I went there, but not sure where I can find the Skill ID.
Looking to get the Skill ID for HondaLink (https://alexa.amazon.com/spa/index.html#skills/dp/B08K879WHF). As I am currently using custom commands and using "ask honda to start my car". However then the Echo wants me to say the PIN.
(Though using the skill directly would probably have the same effect I'm assuming. Is there any way to give the command and auto-follow it up with the PIN code? Right now I am just using a button:)

type: button
name: Start car
show_state: false
tap_action:
  action: call-service
  service: media_player.play_media
  service_data:
    entity_id: media_player.echo_dot
    media_content_type: custom
    media_content_id: ask honda start my car

**Comment by JvdMaat** [comment, marked as answer]
Just to give an update in case anyone else is looking for something similar. I worked around this by creating a separate button for the passcode and just having the media_content_id be '1111' (or whatever passcode you've set).
So click the remote start/unlock/etc button, wait for the Echo Dot to ask for the code, and then click the passcode button.
I may at some point figure out how to code it into a single button with an appropriate delay. But for now this works.

**Comment by thenoid** [comment]
Did you ever find out how to get the skillid?

**Reply by JvdMaat** [reply]
Nope. Still using just the button. Though am planning on swapping it to Node Red and using a delay so I can give two commands in a row. Just need to get some time to sit down and do it.

**Comment by AngelManiac** [comment]
Did you ever get this working? Trying to do something similar in Home Assistant, but the echo is not listening to the PIN (2nd command).

**Reply by JvdMaat** [reply]
Hrm.. I don't have my car right now (Wife took it to work). But it seems like it's not working anymore indeed. I tried to remotely unlock the car, and I hear the Echo Dot asking for the pin, but after I send the pin through the next button, I don't get a confirmation, and in the app, even after refreshing a few times, the car remains locked. So looks like they somehow removed this capability.
I can still get informational data (how much fuel left, etc), But no remote control.

**Reply by AngelManiac** [reply]
Darn. Thanks for speedy reply. Just trying to make sure I'm not crazy. Wasted too much time on it.

**Reply by JvdMaat** [reply]
Yeah, sorry. I really haven't been using the HA Honda commands as they were too much hassle. Send feedback in the Hondalink app/website and request an open API. (Oh, and complain about the myQ Garage opener ad in the Android app that cannot be dismissed. We're already paying for remote functionality, no need for them to ad bomb us with ads that take up half the screen and cannot be removed)
The more requests they get, the more they'll pay attention.

**Reply by AngelManiac** [reply]
I use CarLink w/ Alexa. I assume it's similar to the Honda interface. I've ditched the myQ setup in favor of Meross. I still use the hardware as backup though. Migrating to Meross wasn't difficult and was worth it. I've connected it to HA, Google Assistant, and Alexa. All work.
